src/CMakeLists.txt

```diff
@@ -550,80 +597,34 @@ set_target_properties(${src_executable_gtests}
 add_custom_target(tests_unit
                   COMMAND ${src_executable_gtests}
                   DEPENDS ${src_executable_gtests}
+                  COMMENT "Executing test ${src_executable_gtests}"
                   VERBATIM)
-add_dependencies(${src_executable_gtests} ${src_lib_static_gtest} ${src_common_test_exe_libs})
 
 #
 # tests_all just calls other targets
 #
+# TODO This doesn't seem to have any effect; it's probably because the DEPENDS
+# of add_custom_target must be files, not other high-level targets. If really
+# need to run these tests during build, then either the individual
+# add_custom_target of the individual test runners should be declared with the
+# ALL option, or tests_all target whould be declared without DEPENDS, and
+# add_dependencies should be used to set it's dependencies on the custom targets
+# of the inidividual test runners.
 add_custom_target(tests_all
                   DEPENDS tests_cpp_region
                   DEPENDS tests_unit
                   COMMENT "Running all tests"
                   VERBATIM)
 
-# Dependencies of the combined nupic.core static lib
```

Comment:
This all got replaced by the new function `MERGE_STATIC_LIBRARIES` in NupicLibraryUtils.cmake